receiver/datadogreceiver/internal/translator/series.go

```diff
@@ -52,7 +23,7 @@ type SeriesList struct {
 	Series []datadogV1.Series `json:"series"`
 }
 
-func (mt *MetricsTranslator) translateMetricsV1(series SeriesList) pmetric.Metrics {
+func (mt *MetricsTranslator) TranslateSeriesV1(series SeriesList) pmetric.Metrics {
 	bt := newBatcher()
```

Comment:
Over time the same batchers will be created again and again. **This is not for this PR** but I wonder if an improvement for the receiver would be to have a pool of the maps we've seen in the past and reuse them instead of reallocating them on every request handling.